### PR TITLE
Refactor getFeedbackFromTasks

### PR DESCRIPTION
**Pull request by eatyourgreens**
Refactor getFeedbackFromTasks for legibility.
Remove lodash in favour of Object.entries(tasks) to iterate over workflow tasks and create a feedback rules object.

Package:
lib-classifier

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && yarn bootstrap` and app works as expected?
- [ ] Can you run a [production build](https://github.com/zooniverse/front-end-monorepo#getting-started) of the app?

### CODE REVIEW

File: packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-    if (_.get(task, 'feedback.enabled', false) &&
-      _.get(task, 'feedback.rules', []).length > 0) {
-      return _.assign({}, result, {
-        [taskId]: task.feedback.rules
-      })
-    } else {
-      return result
+function getFeedbackFromTasks (tasks = {}) {
+  const result = {}
+  Object.entries(tasks).forEach(function getTaskRules ([taskKey, task]) {
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js]: If we want to support IE11, then this should be changed to use `Object.keys()`.

Comment (srallen) [reply packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js]: We're not supporting IE11 in the rebuild: https://github.com/zooniverse/front-end-monorepo/blob/master/docs/arch/adr-4.md

Comment (srallen) [reply packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js]: And fwiw, Microsoft is dropping legacy support for it in 2020 and will make Edge available for Win 7 users to download.

File: packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+    const { feedback } = task
+    const taskHasFeedback =
+      feedback &&
+      feedback.enabled &&
+      feedback.rules.length > 0
```
Comment (srallen) [inline packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js]: Maybe `feedback.rules && feedback.rules.length > 0`?

Comment (eatyourgreens) [reply packages/lib-classifier/src/store/feedback/helpers/get-feedback-from-tasks.js]: I've updated to add a check for `rules` and a default value of `{ enabled: false, rules: [] }` for the feedback object.

**Pull request merged**